### PR TITLE
Shenli/show grants

### PR DESCRIPTION
**Pull request by shenli**
Support show grants statement. See: https://dev.mysql.com/doc/refman/5.7/en/show-grants.html
We can start PhpMyAdmin with any error log now.

### CODE REVIEW

**Comment by siddontang** [comment]
remove .testutil_test.go.swp .testutil_test.swp

File: util/testutil/testutil.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+// CompareUnorderedStringSlice compare two string slices.
+// If a and b is exactly the same except the order, it returns true.
+// In otherwise return false.
+func CompareUnorderedStringSlice(a []string, b []string) bool {
+	if len(a) != len(b) {
```
Comment (siddontang) [inline util/testutil/testutil.go]: I think sort and check may be better.

File: util/testutil/testutil_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+package testutil
+
+import (
+	"testing"
+
+	. "github.com/pingcap/check"
+)
+
+const (
```
Comment (shenli) [inline util/testutil/testutil_test.go]: Remove those useless code.

File: util/testutil/testutil.go (hunk 1 above)
Comment (shenli) [inline util/testutil/testutil.go]: May be. But no significant difference.

**Comment by siddontang** [comment]
LGTM

File: util/testutil/testutil_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		{[]string{"1", "1", "2"}, []string{"1", "1", "2"}, true},
+		{[]string{"1", "1", "2"}, []string{"1", "2", "1"}, true},
+		{[]string{"1", "1"}, []string{"1", "2", "1"}, false},
+		{[]string{"1", "1", "2"}, []string{"1", "2", "2"}, false},
+		{nil, nil, true},
+		{[]string{}, nil, false},
+		{nil, []string{}, false},
+	}
+	for _, t := range tbl {
+		c.Assert(CompareUnorderedStringSlice(t.a, t.b), Equals, t.r)
```
Comment (qiuyesuifeng) [inline util/testutil/testutil_test.go]: Can use DeepEquals for this checking?

Comment (shenli) [inline util/testutil/testutil_test.go]: Elements in a and b may have different order.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**